modules/processing/cuckooml.py

```diff
@@ -797,6 +804,14 @@ def filter_dataset(self, dataset=None, feature_coverage=0.1,
 
     def detect_abnormal_behaviour(self, count_dataset=None, figures=True):
         """Detect samples that behave significantly different than others."""
+        
+        # Safety check for plotting
+        if not Config("cuckooml").cuckooml.plotting and figures:
+            print >> sys.stderr, "Warning: 'plotting' and 'figures' do not match. \
+                                  Plotting modules might not be imported."
```

Comment:
Meaningful error message would be nice here as well. It should tell the user that plotting is disabled (globally) in config file and that the local plotting flag will not have any effect.

Also, one new line is enough.